app/api/internal/stats.rb

```diff
@@ -6,6 +6,9 @@ module API
           respond_with_404('service not found') unless Service.exists?(params[:service_id])
         end
 
+        # This is very slow and needs to be disabled until the performance
+        # issues are solved. In the meanwhile, the job will just return OK.
```

Comment:
> In the meanwhile, the job will just return OK

Good for me then 😜

Reply:
Yes, it's done this way to avoid breaking compatibility with the latest version of Pisoni.